package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "serverless-provisioned-concurrency-autoscaling",
-  "version": "1.8.0",
+  "version": "1.7.0",
```

Comment:
Should this be 1.9.0?

Reply:
this is package-lock.json
but let me run `npm i` + tests locally to update this file